Juliana/second-assignment/styles.css

```diff
@@ -0,0 +1,368 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap');
+*{
+    margin: 0px;
+    padding: 0px;
+    box-sizing: border-box;
+    font-family: 'Roboto';
+}
+a{
+    text-decoration: none;
+}
+body{
+    width: 100%;
+}
+header{
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: 100%;
+    height: 8vh;
+    background-color: #0A0A0A;
+}
+.open-menu,
+.close-menu{
+    display: none; 
+} 
+.nav-links{
+    list-style-type: none;
+    display: flex;
+    gap: 1rem;
+    margin-left: 1.5rem;
+}
+    .nav-links li a{
+        position: relative;
+        overflow: hidden;
+        display: inline-block;
+        font-size: 1rem;
+        font-weight: 600;
+        padding: 1rem;
+        color: whitesmoke;
+        text-shadow: 0 0 .5rem #333;
+    }
+    .nav-links a:after{
+        content: "";
+        position: absolute;    
+        bottom: 0;
+        left: -100%;
+        width: 100%;
+        height: 2px;
+        background: red;  
+        transition: left .6s;    
+    }
+    .nav-links a:hover:after{
+        left: 0;
+    }
+    .nav-links a:hover{   
+        color: red;
+        transition: .3s ease-in;
+    } 
+#bg-img{ 
+    width: 100%;
+    height: 92vh;
+    position: absolute;
+    object-fit: cover;
+    object-position: center;
+}
+#intro div{
+    position: relative;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    margin-left: 30px;
+    padding-top: 80px;
+    width: 500px;
+    height: 92vh;
+    max-width: 80%;
+}
+#intro h1,p{
+    display: flex;
```

Comment:
Normally, we leave flex properties for presentational containers (divs and the semantic ones), and just let the flex properties of those parent containers determine the alignment of their flexbox children. #intro div could have a align-items: center; property to align its content

Reply:
I've included the flex properties as suggested in commit [0876d3b]